signer/src/transaction_signer.rs

```diff
@@ -514,7 +504,7 @@ where
                 ..
             }) => {
                 tracing::info!("DKG ended in failure: {fail:?}");
```

Comment:
Nothing you did here but just noticed that this should probably be an error/warn.